examples/virtio/virtio.mk

```diff
@@ -0,0 +1,139 @@
+QEMU := qemu-system-aarch64
+
+VMM_TOOLS := $(LIBVMM)/tools
+MICROKIT_TOOL ?= $(MICROKIT_SDK)/bin/microkit
+SDDF_INCLUDE := $(SDDF)/include/sddf
+UTIL := $(SDDF)/util
+
+UART_DRIVER := $(SDDF)/drivers/serial/$(UART_DRIVER)
+SERIAL_COMPONENTS := $(SDDF)/serial/components
+BLK_COMPONENTS := $(SDDF)/blk/components
+
+SERIAL_NUM_CLIENTS := -DSERIAL_NUM_CLIENTS=3
```

Comment:
Put this in the header to be consistent with sDDF.

Reply:
Done